newdle/core/util.py

```diff
@@ -1,14 +1,20 @@
 import re
 from datetime import datetime, time
 from enum import Enum
+from io import BytesIO
 
-from flask import current_app
-from itsdangerous import Signer
+from flask import current_app, render_template, send_file
+from itsdangerous import Signer, URLSafeSerializer
+from werkzeug.local import LocalProxy
 
 
 DATE_FORMAT = '%Y-%m-%d'
 DATETIME_FORMAT = '%Y-%m-%dT%H:%M'
 
+secure_serializer = LocalProxy(
+    lambda: URLSafeSerializer(current_app.config['SECRET_KEY'], b'newdle')
+)
```

Comment:
What about moving the one from auth.py in here as well (and renaming it to `secure_timed_serializer` to make clear that we have to different ones?

Reply:
yeah, actually did that locally but didn't push it